src/media-popup.js

```diff
@@ -54,4 +71,4 @@ ipcRenderer.on('load-media', (event, mediaUrl, mediaType) => {
     video.autoplay = true;
     video.playbackRate = 1;
   }
-});
+});
```

Comment:
> `});` is bad

This is posix rule.
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

> `});\n\n` is bad too

This is my style. Isn't it just lengthy?